templates/typescript_gapic/test/gapic-$service-$version.ts.njk

```diff
@@ -356,5 +356,29 @@ describe('{{ service.name }}Client', () => {
             });
         });
     });
+    describe('{{ method.name.toCamelCase() }}Stream', () => {
```

Comment:
Thank you for adding the unit test for this method!

Reply:
:) thanks!